src/IROperator.cpp

```diff
@@ -1136,6 +1136,20 @@ Expr unsafe_promise_clamped(const Expr &value, const Expr &min, const Expr &max)
                                 Internal::Call::Intrinsic);
 }
 
+namespace Internal {
+Expr promise_clamped(const Expr &value, const Expr &min, const Expr &max) {
```

Comment:
Worth adding a comment emphasizing that this is really really really not for external use